server/src/resolvers/Mutation.ts

```diff
@@ -1,30 +1,31 @@
 import bcrypt from 'bcryptjs'
 import  jwt  from "jsonwebtoken"
-import {  APP_SECRET, getUserId } from "../utils.js"
+import {  APP_SECRET } from "../utils"
+import { PrismaClient } from "@prisma/client";
 
 
 export const Mutation = {
 
-    addBlog: async (parent, { input }, context) => {
+    addBlog: async (parent: any, {input} : { input:{ title:string, body:string, small:string, name:string} }, context:any) => {
 
-        const  { title, body, small } = input;
+        const  { title, body, small, name } = input;
 
         const { userId } = context;
 
         const addBlog = await context.prisma.blog.create({
-/*             include:{author:true}, */
             data: { 
                 title: title,
                 body:body,
                 small:small,
+                name:name,
                 author: { connect: { id: userId } },
             }
```

Comment:
If key and value are the same, it is enough to add it once: 

```suggestion
            data: { 
                title,
                body,
                small,
                name,
                author: { connect: { id: userId } },
            }
```